generation_config.yaml

```diff
@@ -16,7 +16,7 @@ libraries:
       codeowner_team: '@googleapis/api-bigquery'
       api_id: bigquerystorage.googleapis.com
       transport: grpc
-      requires_billing: true
+      requires_billing: true 
```

Comment:
Unintentional extra white space.

Reply:
I added that whitespace to confirm the generation goes well. I'll remove it now.